### ISSUE TITLE
Analysis without other submissions

### ISSUE THREAD

**Issue by dmvt** [code-423n4/org#142]
I propose a change to only allow the submission of an advanced-analysis if at least 1 high or medium risk issue has been submitted.

**Comment by bytes032** [comment]
Agreed. Otherwise, it turns into chatbot driven analysis reward pot farming.

**Comment by 0xSorryNotSorry** [comment]
Majority of the analysis reports don't add anything to the Sponsors as the reports only provide functionality features, summary of the contracts etc. I definitely agree with that there should be an access control to submit analysis reports.

**Comment by 0xean** [comment]
@dmvt is the requirement that they are valid H/Ms ?

**Comment by dmvt** [comment]
No necessarily. More that effort was put into them.

**Comment by 0xean** [comment]
> Not necessarily. More that effort was put into them.

so this adds another hoop a judge has to jump through to manually validate? IE is this H/M satisfactory enough in order for me to consider their Analysis report?

**Comment by CloudEllie** [comment]
@dmvt from a tooling perspective, an option that might be simpler to implement would be to restrict access to the Analysis submission form to only wardens who have at least one valid HM on the C4 leaderboard. (This could be all-time or time-constrained.) 

What do you think of that option, vs submitting a well-argued HM to the same audit?

**Comment by MarioPoneder** [comment]
What about both?
To get the Analysis submission form activated one needs:
* A valid HM on the 365 day leaderboard
* A HM submission in the current contest

**Comment by 14si2o** [comment]
I've also noticed the recent increase in professional chatbot wardens who only submit Analyses. 

Perhaps the simplest solution would be to limit Analysis submission to Wardens with backstage access? 

- Only people who actually submitted valid findings in the past can participate
- No extra work for pre-sorters or judges
- Should 90%+ of the chatbot spam I hope
